beoordelingstool_dockwidget.py

```diff
@@ -895,18 +916,103 @@ def save_zip_to_server(project_name, temp_dir, zip_url, user_data):
         (str) zip_url: The url to save the zip to.
         (dict) user_data: A dict containing the username and password
     """
+    # Check user login credentials ()  # not needed, checked when json is uploaded
+    username = user_data["username"]
+    password = user_data["password"]
     if zip_url is None:
         iface.messageBar().pushMessage("Error", "The json has no url.", level=QgsMessageBar.CRITICAL, duration=0)
         return
     else:
-        # Add error handling
-        encoded_user = base64.b64encode(user_data)
-        # Put the key 'Upload reviews' in the POST request
-        # Give a key to the url that shows that a zip is uploaded
-        data = {
-            'shape_files': open(os.path.join(temp_dir, "{}.zip".format(project_name))).read(),
-            'Upload reviews': ''
-        }
-        req = urllib2.Request(zip_url, data, encoded_user)
-        response = urllib2.urlopen(req)
-        the_page = reponse.read()  # nodig
+        manholes_layerList = QgsMapLayerRegistry.instance().mapLayersByName(SHP_NAME_MANHOLES)
+        pipes_layerList = QgsMapLayerRegistry.instance().mapLayersByName(SHP_NAME_PIPES)
+        measuring_points_layerList = QgsMapLayerRegistry.instance().mapLayersByName(SHP_NAME_MEASURING_POINTS)
+        if manholes_layerList and pipes_layerList and measuring_points_layerList:
+            # Get project name from the json saved in the same folder as the "manholes" layer
+            layer_dir = get_layer_dir(manholes_layerList[0])
+            zip_path = os.path.join(layer_dir, "{}.zip".format(project_name))
+
+            form = MultiPartForm()
+            filename = os.path.basename(zip_path)
+            form.add_field('Upload reviews', 'Upload reviews')
```

Comment:
Hier gaat het denk ik fout: dit veld is nodig voor de json upload, voor de zip upload heet het veld anders.